entrypoint.sh

```diff
@@ -23,9 +24,10 @@ INPUT_WORKDIR="${11}"
 #
 REPOSITORY="${INPUT_REPOSITORY:-${GITHUB_REPOSITORY}}"
 BRANCH="${INPUT_BRANCH}"
-HOST="${INPUT_GIT_HOST:-github.com}"
+HOST="${INPUT_HOST:-github.com}"
 TOKEN="${INPUT_GITHUB_PAT:-${INPUT_GITHUB_TOKEN}}"
 REMOTE="${INPUT_REMOTE:-https://${TOKEN}@${HOST}/${REPOSITORY}.git}"
+REMOTE_WITHOUT_TOKEN="https://${HOST}/${REPOSITORY}.git"
```

Comment:
Maybe these should be swapped. e.g. `REMOTE` lacks the token and we add a new `CREDENTIALED_REMOTE` that includes the token?